### DISCUSSION TITLE
Release 2.0.0: new Ansible core and python requirements on server

### DISCUSSION THREAD

**Discussion by robgjertsen1** [IBM/ansible-for-i#173] [Announcements]
Collection release 2.0.0 is essentially an update to require newer ansible-core version on the control node in order to comply with certification requirements along with ansible-lint code clean up (still not done) that also was necessary.
Please note the newer levels for ansible-core and python needed for the ansible server / control node below before you install this level (can stay at just published 1.9.2 if don't meet the requirements yet).

CHANGELOG:

v2.0.0 (2023-11-22)

With this release the collection now requires ansible-core version 2.14 or 2.15 on the Ansible server / control node. This also requires having Python 3.9 on the Ansible server / control node. The IBM i target nodes may still run at a lower level of Python 3. These changes are necessary for continued Redhat certification of the collection and also for Ansible Galaxy.
Bug Fixes

    Fixes for ansible 2.14 and 2.15 (primarily sanity test related).
    Fix present_ip_interface role which was broken due to some incorrect var names defined in defaults/main.yml.

Miscellaneous

    Modifications for ansible-lint in production mode that are a work in progress for the collection (rest of changes in following release).
        Completely clean up playbooks.
        Extensive clean up of roles (but additional work required to change role variables / API for all roles).

Documentation

    Document new requirements for ansible-core version 2.14+ and python version 3.9. Add new Ansible install instructions for an IBM i Ansible server.
    Add CHANGELOG file

**Comment by K2701** [comment]
I run Ansible Controller on the IBM i in PASE and have a “chroot” environment setup to keep build configurations isolated for testing.

I see this release should work with 2.15.0 which I have loaded in that environment

======================================= 
xxxxxx[@isdev:-bash-5.1-~$](xxxxxx@xxxxx-bash-5.1-~$) chroot /QOpenSys/containers/kcontainer22 /QOpenSys/usr/bin/sh
u@h:s-v-w$ ansible --version
ansible [core 2.15.0] 
  config file = None
  configured module search path = ['/home/xxxxxx/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /QOpenSys/pkgs/lib/python3.9/site-packages/ansible
  ansible collection location = /home/xxxxxx/.ansible/collections:/usr/share/ansible/collections
  executable location = /QOpenSys/pkgs/bin/ansible
  python version = 3.9.16 (main, Feb  1 2023, 13:01:20) [GCC 6.3.0] (/QOpenSys/pkgs/bin/python3)
  jinja version = 3.1.2
  libyaml = False
u@h:s-v-w$

======================================= 

I’ll try it with this new update 1.9.2 and see how it performs.
I look forward to all the updates!

Thank You
